Modules/TOF/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ install(FILES tofdigits.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_MC.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct_MC.json
+	      tofMatchedTracks_ITSTPCTOF_TPCTOF_ITSTPCTRDTOF_direct_MC.json
```

Comment:
Why there is ITS-TPC-TRD-TOF and  not TPC-TRD-TOF?

Reply:
A typo while I tried to add TRD back, thanks for spotting (there was also TPC-TRD missing)